client/go/outline/config/module.go

```diff
@@ -98,6 +98,17 @@ func NewDefaultTransportProvider(tcpDialer transport.StreamDialer, udpDialer tra
 	packetEndpoints.RegisterSubParser("websocket", NewWebsocketPacketEndpointSubParser(streamEndpoints.Parse))
 
 	// Stream dialers.
+	streamDialers.RegisterSubParser("iptable", func(ctx context.Context, input map[string]any) (*Dialer[transport.StreamConn], error) {
```

Comment:
Please move this to config_iptable like we do for the other subtypes to keep the module code clean.